### ISSUE TITLE
(BUG) TypeError: Cannot read property 'roomId' of undefined

### ISSUE THREAD

**Issue by cinarizasyon** [antoine92190/vue-advanced-chat#14]
## Describe the bug
I just imported the library to a new vue-cli project and tried to run with adding example rooms data but got this error. All changes are in App.vue file.

App.Vue:

```html
<template>
  <div id="app">
    <chat-window :currentUserId="currentUserId" :rooms="rooms" :messages="messages" />
  </div>
</template>

<script>
import ChatWindow from "vue-advanced-chat";
import "vue-advanced-chat/dist/vue-advanced-chat.css";
export default {
  name: "App",
  components: { ChatWindow },
  data() {
    return {
      messages: [],
      currentUserId: 1234
    };
  },
  computed: {
    rooms() {
      return [
        {
          roomId: "1",
          roomName: "Room 1",
          lastMessage: {
            content: "Last message received",
            sender_id: 1234,
            username: "John Doe",
            timestamp: "10:20",
            date: 123242424,
            seen: false,
            new: true
          },
          users: [
            {
              _id: 1234,
              username: "John Doe",
              status: {
                state: "online",
                last_changed: "today, 14:30"
              }
            },
            {
              _id: 4321,
              username: "John Snow",
              status: {
                state: "offline",
                last_changed: "14 July, 20:00"
              }
            }
          ],
          typingUsers: [4321]
        }
      ];
    }
  }
};
</script>

<style>
#app {
  font-family: Avenir, Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  margin-top: 60px;
}
</style>

```
package.json:

```json
{
  "name": "message-to-world",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "lint": "vue-cli-service lint"
  },
  "dependencies": {
    "core-js": "^3.6.5",
    "vue": "^2.6.11",
    "vue-advanced-chat": "^0.4.2"
  },
  "devDependencies": {
    "@vue/cli-plugin-babel": "^4.4.0",
    "@vue/cli-plugin-eslint": "^4.4.0",
    "@vue/cli-service": "^4.4.0",
    "babel-eslint": "^10.1.0",
    "eslint": "^6.7.2",
    "eslint-plugin-vue": "^6.2.2",
    "vue-template-compiler": "^2.6.11"
  },
  "eslintConfig": {
    "root": true,
    "env": {
      "node": true
    },
    "extends": [
      "plugin:vue/essential",
      "eslint:recommended"
    ],
    "parserOptions": {
      "parser": "babel-eslint"
    },
    "rules": {}
  },
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "not dead"
  ]
}

```

## Screenshots

![image](https://user-images.githubusercontent.com/53943169/83338649-23286000-a2cf-11ea-9854-9378b6a4e218.png)

## Device

- OS: MacOS Mojave
- Browser: Chrome 83
- Package version: 0.4.2
- Vue version: 2.6.11

**Comment by antoine92190** [comment]
@cinarizasyon Fixed in 52f1ebdfaf148efcad175dea35eddff05d89b3fc

**Comment by cinarizasyon** [comment]
Thank you so much!
